src/main/kotlin/com/exactpro/th2/ws/client/Main.kt

```diff
@@ -152,7 +154,7 @@ fun run(
                 require(messagesCount == 1) { "Message group contains more than 1 message" }
                 val message = messagesList[0]
                 require(message.hasRawMessage()) { "Message in the group is not a raw message" }
-                settings.frameType.send(client, message.rawMessage.body.toByteArray())
+                settings.frameType.send(client, message.rawMessage.body.toByteArray(), message.rawMessage.parentEventId)
```

Comment:
> message.rawMessage.parentEventId

It'll always be non-null. We should check its presence through `hasParentEventId` method instead and pass `null` if it returns `false`